packages/package_bumbershoot_3_0_8738/.shed.yml

```diff
@@ -1,11 +0,0 @@
-categories:
-- Tool Dependency Packages
-description: Contains a tool dependency definition that downloads the Bumbershoot tools.
-long_description: |
```

Comment:
I would keep them for historical reasons. But if you are sure they never worked ... you decide ;)